screensaver.turnoff/screensaver.py

```diff
@@ -0,0 +1,164 @@
+import sys
+import subprocess
+
+import xbmc
+import xbmcaddon
+import xbmcgui
+
+
+def _log_error(msg='', level=xbmc.LOGERROR):
+    xbmc.log(msg='[%s] %s' % (addon_name, msg), level=level)
+
+
+def _log_notice(msg='', level=xbmc.LOGNOTICE):
+    xbmc.log(msg='[%s] %s' % (addon_name, msg), level=level)
+
+
+def _popup(title='', msg='', delay=10000, image=''):
+    if not title:
+        title = '%s screensaver failed' % addon_name
+    if not image:
+        image = addon_icon
+    xbmc.executebuiltin('XBMC.Notification("%s","%s",%d,"%s")' % (title, msg, delay, image))
```

Comment:
please use `xbmcgui.Dialog().notification()` instead of the builtin:

https://codedocs.xyz/xbmc/xbmc/group__python___dialog.html#ga27157f98dddb21b44cc6456137977aa2

Reply:
Great, I didn't know. Followed other examples here. Fixed now.